Sources/StreamChatUI/ChatMessageList/ChatMessageListVCDataSource.swift

```diff
@@ -7,6 +7,9 @@ import StreamChat
 
 /// The object that acts as the data source of the message list.
 public protocol ChatMessageListVCDataSource: AnyObject {
+    /// Asks the data source to return all the available messages.
+    var messages: [ChatMessage] { get }
+
```

Comment:
I find this change useful in terms of data source protocol consistency but are we using it somewhere in diffing logic?

Reply:
This is needed when adding the initial data. Because we open a channel with the local data already, we don't get notified of all the messages available through the `updateMessages` event. So we need to get the messages from the channel controller. 

This is used here: https://github.com/GetStream/stream-chat-swift/pull/1770/files/98fc37e0e04f6954a1f7ff4c139e3ef367b41dff#diff-30faf0001fbdda5a68aa7eca1fe1d37eeef9053d44f81afc1c6365ed66f44869R630